src/app/pages/give/give.js

```diff
@@ -202,6 +214,12 @@ export default class Give extends CMSPageController {
         if (invalid) {
             event.preventDefault();
             this.update();
+        } else {
+            localStorage.setItem('donation:first_name', this.el.querySelector('[name=First_Name]').value);
```

Comment:
It's the names in the querySelector need to be the same as the name attributes in the HTML form. You have
`<input type="hidden" name="first_name">`
but are querying
`this.el.querySelector('[name=First_Name]')`